faunadb-java-dsl/src/main/java/com/faunadb/client/HttpResponses.java

```diff
@@ -121,17 +122,17 @@ public String description() {
     private final String description;
     private final ImmutableList<ValidationFailure> failures;
 
-    public QueryError(ImmutableList<String> position,
+    public QueryError(List<String> position,
                       String code,
                       String description,
                       ImmutableList<ValidationFailure> failures) {
-      this.position = position;
+      this.position = ImmutableList.copyOf(position);
```

Comment:
are you dropping `google.common` entirely or just public usage? What about `Collections.unmodifiableList()`